crates/egui/src/ui.rs

```diff
@@ -1165,6 +1174,67 @@ impl Ui {
     pub fn ui_contains_pointer(&self) -> bool {
         self.rect_contains_pointer(self.min_rect())
     }
+
+    /// Find and close the first closable parent.
+    /// Use [`UiBuilder::closable`] to make a [`Ui`] closable.
+    /// You can then use [`Ui::should_close`] to check if it should be closed.
```

Comment:
We should always keep the first paragraph short in a docstring, as that is the summary that will show up in the overview in cargo doc (see with `cargo doc -p egui --open`)

```suggestion
    /// Find and close the first closable parent.
    ///
    /// Use [`UiBuilder::closable`] to make a [`Ui`] closable.
    /// You can then use [`Ui::should_close`] to check if it should be closed.
```